### ISSUE TITLE
Items disappear when using "/rpginv reload"

### ISSUE THREAD

**Issue by KubosKube** [EndlessCodeGroup/RPGInventory#71]
I can give flowers to players through "/rpginv item" and they stack just fine, but when I use "/rpginv reload", any such items being held within the player's vanilla inventory or their RPG inventory will be turned into single items.

On a slightly related issue, RPG inventory items that have been manually enchanted(i.e. boots with Frost Walker) will lose their enchantments.

**Comment by OsipXD** [comment]
Can you provide me item configuration?

**Comment by KubosKube** [comment]
items.yml
```items:
  test-item:
    texture: INK_SACK:1
    name: "Test Item"
    lore:
    - "&b&oTest item for test purposes."
    rarity: UNCOMMON
```

slots.yml
```slots:
  test:
    type: PASSIVE
    slot: [42, 43, 44, 51, 52, 53]
    holder:
      item: DIAMOND_HOE:16
      name: "&a&lTest Slot"
      lore:
      - "&6&oTest items go here for testing purposes."
    items:
      - INK_SACK:1
```


[RPG Debugging.zip](https://github.com/EndlessCodeGroup/RPGInventory/files/1342943/RPG.Debugging.zip)

**Comment by OsipXD** [comment]
Thanks, I'll check it

**Comment by OsipXD** [comment]
Hello! Sorry for long absence.
I've tried to reproduce your bug, but it works fine for me.

**My steps:**
1. Give two test-items to player
2. Place it into test-slot
3. Use `/rpginv reload`
4. And.. nothing happens

**Comment by KubosKube** [comment]
Hello again. Very sorry I took so long to get back to you.
I've updated my server to RPG Inventory to 2.1.4 since we last spoke, and I'm using the newest versions of spigot and craftbukkit.


Here's a video:
https://youtu.be/QpmZhQWWfdQ

And here's a download for my entire server setup (Just craftbukkit/spigot jars, RPG Inventory, Vault, Protocol Lib):
[File in Google Drive](https://drive.google.com/file/d/1kxZOYp475VC1sUaLgmPDOhP26o3aIrbF/view?usp=sharing)
